crates/base-db/src/lib.rs

```diff
@@ -57,7 +57,10 @@ pub struct Files {
 
 impl Files {
     pub fn file_text(&self, file_id: vfs::FileId) -> FileText {
-        *self.files.get(&file_id).expect("Unable to fetch file; this is a bug")
+        *self
+            .files
+            .get(&file_id)
+            .expect("Unable to fetch file text for `vfs::FileId` {id}; this is a bug")
```

Comment:
`expect()` doesn't accept formatting args?

Reply:
whoops, I'm a dummy. completely forgot!

Reply:
I think I wrote this in a hurry + didn't have semantic highlighting to catch this mistake... whoops.